WordPress/Classes/Services/MediaImageService.swift

```diff
@@ -1,7 +1,8 @@
-import UIKit
+import AsyncImageKit
 import CoreData
+import UIKit
+import WordPressData
 import WordPressShared
-import AsyncImageKit
```

Comment:
In hindsight, sorting the imports alphabetically in the context of this PR makes the diff harder to read. Apologies.